odoo/fields.py

```diff
@@ -1765,7 +1763,12 @@ def _get_stored_translations(self, record):
         # assert (self.translate and self.store and record)
         record.flush_recordset([self.name])
         cr = record.env.cr
-        cr.execute(f'SELECT "{self.name}" FROM "{record._table}" WHERE id = %s', (record.id,))
+        cr.execute(SQL(
+            "SELECT %s FROM %s WHERE id = %s",
+            SQL.identifier(self.name),
+            SQL.identifier(record._table),
+            record.id,
+        ))
```

Comment:
I have to say that it is kind of confusing to have safe and unsafe (before casting) arguments in the same place.
It's kind of easy to miss during a review too. (whereas before the query was unsafe and params were safe, easier to divide in my mind at least)

I would not be surprised if people used `SQL` like they shouldn't, and I don't think we want to ping security every time `SQL` is used, like for `Markup`. Or do we?

Reply:
`SQL(string, ...)` is safe when `string` is a literal, and we will add a linter to warn when `string` is not a literal.
On the other hand, `SQL.identifier(x)` is _always_ safe, since its result is guaranteed to be properly escaped (or raise an exception if escaping is not guaranteed).

Reply:
I agree then :+1: 
Would be nice to have the same linter for `Markup`s of literals though, to avoid bothering the security team too much